source/Calamari.Shared/Integration/Packages/NuGet/NuGetV3LibDownloader.cs

```diff
@@ -46,6 +47,7 @@ public static void DownloadPackage(string packageId, IVersion version, Uri feedU
             }
 
             File.Move(targetTempNupkg, targetFilePath);
+            sourceCacheContext.Dispose();
```

Comment:
Since `SourceCacheContext` implements `IDisposable`, could we use a `using` for it? This ensures the temp folder is deleted even exceptions were thrown in the middle.